qcodes/instrument_drivers/rohde_schwarz/ZNB.py

```diff
@@ -447,10 +468,80 @@ def __init__(self, name: str, address: str, init_s_params: bool=True, **kwargs)
             self.display_sij_split()
             self.channels.autoscale()
 
-        self.update_display_on()
-        self.rf_off()
+        self.display_updates('ON')
+        self.rf_power(False)
         self.connect_message()
 
+    def rf_on(self) -> None:
+        warn("The function rf_on has been deprecated. Please use the "
```

Comment:
why didn't you used the `@deprecate` decorator from qcodes?

Reply:
I totes forgot about it until I was halfway done... Do you think it matters now?

Reply:
it almost doesn't matter but there's this little thing that `@deprecate` uses `DeprecationWarning`: `warnings.warn(msg, DeprecationWarning, ...)`. I think it's better to use `DeprecationWarning`. And for that matter, changing to using `@deprecate` is probably the same amount of effort.